examples/inner_html/src/lib.rs

```diff
@@ -4,8 +4,12 @@ use web_sys::{console, Node};
 use yew::virtual_dom::VNode;
 use yew::{Component, ComponentLink, Html, ShouldRender};
 
-const SVG: &str = r#"
-<h2>Inline SVG or <i>any</i> HTML:</h2>
+const HTML: &str = r#"
```

Comment:
I'd prefer if we used `include_str!` here: https://doc.rust-lang.org/std/macro.include_str.html

Reply:
Done, thanks!